src/members/models.py

```diff
@@ -1,3 +1,30 @@
-# from django.db import models
+"""Database model for this app"""
+import uuid
+from django.db import models
+from django.core.validators import RegexValidator
+from django.contrib.auth.models import AbstractUser
 
-# Create your models here.
+class Member(AbstractUser):
+    """Extended custom user from User"""
+
+    UUID = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    username = models.CharField(max_length=70, unique=True)
+    email = models.CharField(max_length=320, unique=True, null=False, blank=False)
+    password = models.CharField(max_length=120,
+                                validators=[
+                                    RegexValidator(
+                                        regex='^(?=.*[a-z])(?=.*[A-Z])'
+                                              '(?=.*\\d)(?=.*[@$!%*?&])'
+                                              '[A-Za-z\\d@$!%*?&]{8,}$',
```

Comment:
I'm not sure this validator works for the input through the website, since I can use 'testpass' as a password.